src/IssueViz.Security.UnitTests/Hotspots/LocalHotspotStoreTests.cs

```diff
@@ -122,13 +126,44 @@ public void UpdateForFile_NoServerHotspots_AddsForDifferentFile()
 
         testSubject.UpdateForFile("file2", newHotspots);
 
-        VerifyContent(testSubject,
-            new LocalHotspot(issueVis1),
-            new LocalHotspot(issueVis2));
+        VerifyContent(testSubject, 
+            new LocalHotspot(issueVis1, default),
+            new LocalHotspot(issueVis2, default));
         eventListener.Events.Should().HaveCount(2).And.Subject.Last().Should()
             .BeEquivalentTo(new IssuesChangedEventArgs(Array.Empty<IAnalysisIssueVisualization>(), newHotspots));
     }
 
+    [TestMethod]
+    public void UpdateForFile_NoServerHotspots_UsesReviewPriority()
```

Comment:
As discussed offline, if this test is just checking the correct review priority is being used then it can be much simpler: it doesn't need to call `RemoveForFile` twice, and it doesn't need to check the events that were raised. Same for the other new test.